tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
```

Comment:
I think we want to add `"declaration": true` here and include `"types": "./dist/evergreen-ui.d.ts" (or similarly named) in `package.json`.

Reply:
Yes! Compiling will be fixed once I'm done with the actual migration, it's in my todo in the OP